extensions/git/package.json

```diff
@@ -1893,6 +1893,25 @@
           "default": [],
           "scope": "resource"
         },
+        "git.branchProtectionIndicator": {
+          "type": "object",
+          "additionalProperties": false,
+          "description": "%config.branchProtectionIndicator%",
+          "properties": {
+            "branchPicker": {
+              "type": "boolean",
+              "description": "%config.branchProtectionIndicator.branchPicker%"
```

Comment:
Maybe `quickOpen` instead of `branchPicker`. The argument is that `statusBar` is a UI element while `branchPicker` is a feature. For consistency, might be better to stick with UI elements only.

Reply:
Fixed. Switched to `quickOpen`.